src/components/game/index.jsx

```diff
@@ -1,9 +1,57 @@
-import React from "react";
+import React, { useState } from "react";
 import Board from "components/game/Board";
 
 // Renders the Board component
 function Game() {
-  return <Board />;
+  const [valuesArr, setValuesArr] = useState(Array(9).fill(null)); // Values of the 1st-9th square
+  const [nextPlayer, setNextPlayer] = useState("O"); // Initial player aka first player to go is O
+  const winner = detectWinner(); // Checks if there is a winner every render
+
+  // To detect winner
+  function detectWinner() {
+    const lines = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [2, 4, 6],
+    ];
+
+    for (let i = 0; i < lines.length; i++) {
+      const [a, b, c] = lines[i];
+      if (valuesArr[a] && valuesArr[a] === valuesArr[b] && valuesArr[a] === valuesArr[c]) {
+        return valuesArr[a];
+      }
+    }
+
+    return null;
+  }
+
+  // To reset board
+  function reset() {
+    let valuesArrClone = valuesArr.slice(); // To create a copy of valuesArr that we can edit
+    valuesArrClone = []; // To clear the copied array
```

Comment:
Top: default `valuesArr`
Bottom: `valuesArr` after clicking reset
![Screen Shot 2021-08-09 at 6 05 54 PM](https://user-images.githubusercontent.com/27680118/128690703-8ea67371-b093-43d6-96d6-b187bb6d37fa.png)

The first line is unnecessary because you overwrite it anyway with the second line. The second line also is not the best value due to the screenshot above (imagine we are rendering the `Square`s by iterating through the array and not hardcoded like what we have right now).

So a better way to do this is: `setValuesArr(Array(9).fill(null));`